nncf/experimental/torch/fx/transformations.py

```diff
@@ -166,8 +197,11 @@ def constant_update_fn(model: torch.fx.GraphModule, node: torch.fx.Node, value:
             f"Constant on input port {input_port_id} for {node} is expected,"
             f" but node {args[input_port_id]} is present."
         )
+    weight_node = args[input_port_id]
+    consumer_nodes = list(weight_node.users.keys())
     args[input_port_id] = new_constant
-    node.args = tuple(args)
+    for node in consumer_nodes:
+        node.replace_input_with(weight_node, new_constant)
```

Comment:
What if I have two nodes which share one constant, and I want to update constant only for the first node? This code will replace constant for both operations

Reply:
ah, good point. Maybe I can add a flag and create another builder which updates constant for all the nodes and calls this update function with the flag set to true.
something like this:
```
def update_const_builder():
    update_const_fn()

def update_const_fn(flag=False):
    #Update constant function

def update_shared_const():
    update_const_fn(flag=True)
```

Reply:
We can try to receive `List[PTTargetPoint]` in the transformation instead of `PTTargetPoint`

Reply:
I'm sorry, I didn't understand. Where do you suggest I change this?